sunpy/image/slit.py

```diff
@@ -0,0 +1,131 @@
+import sunpy.map
+
+import numpy as np
+
+import astropy.units as u
+
+
+def slit(mcube_in, range_a, range_b):
+    """
+    Returns an array with intensity along the slit on the y axis and time
+    along x.
+
+    Parameters
+    ----------
+    mcube_in : `sunpy.map.MapCube`
+        A mapcube of the images you want to perform the slit analysis on.
+        Usually with x and y as space, and z as time.
```

Comment:
You should mention here that each map in the map cube has to be the same resolution and size.